core/modules/startup/story.js

```diff
@@ -183,7 +183,234 @@ function updateLocationHash(options) {
 	var storyList = $tw.wiki.getTiddlerList(DEFAULT_STORY_TITLE),
 		historyList = $tw.wiki.getTiddlerData(DEFAULT_HISTORY_TITLE,[]),
 		targetTiddler = "";
-	if(options.targetTiddler) {
+	if(options.targetTiddler) {/*\
```

Comment:
It seems there is a copy past problem here. In line 187 the source file seems to start again. You need to check that.

Reply:
Wow.  How did I miss that?!!  It was bedtime, and I was tired, but that's a large miss!

I will see if I can update from my phone.  Committing to public GitHub is not allowed here at GigantiCorp.  If not, I'll do it this evening from home.

Reply:
You should do it from home and test properly, prior to submitting a new change.

Reply:
Too late.  I managed it from my phone.  This is bizarre, as it was running fine locally when I committed.  No clue how it was with that mess of a file.

I did some minimal testing on the Vercel preview, but I also can only see that from my phone.  I will check more thoroughly this evening.